src/app/pages/common/entity/entity-table/entity-table-actions.component.scss

```diff
@@ -4,7 +4,6 @@
 
 
 .panel_container {
-    padding-top: 4px;
```

Comment:
This change is part of the PR for Services page. Probably it's better to revert it